modules/cluster-logging-updating-logging.adoc

```diff
@@ -5,27 +5,29 @@
 [id="cluster-logging-updating-logging_{context}"]
 = Updating cluster logging
 
-After updating the {product-title} cluster, you can update cluster logging from 4.3 to 4.4 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.5 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator. You must update both the Elasticsearch Operator and the Cluster Logging Operator at the same time. Kibana is unusable when the Elasticsearch Operator has been upgraded but the Cluster Logging Operator has not been upgraded.
 
-ifdef::openshift-enterprise,openshift-webscale[]
 [IMPORTANT]
 ====
-Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release. You create a ConfigMap to configure *out_forward*. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed.
-
-If you use the *out_forward* plug-in, before updating, you can copy your current `secure-forward.conf` section from the Fluentd ConfigMap and use the copied data when you create the `secure-forward` ConfigMap.
+You must update the Elasticsearch Operator before updating the Cluster Logging Operator. 
 ====
-endif::[]
 
 .Prerequisites
 
-* Update the cluster from 4.3 to 4.4.
+* Update the cluster from 4.5 to 4.5.
 
 * Make sure the cluster logging status is healthy:
 +
 ** All Pods are `ready`.
 ** Elasticsearch cluster is healthy.
 
-* Optionally, copy your current `secure-forward.conf` section from the Fluentd ConfigMap for use if you want to create the `secure-forward` ConfigMap. See the note above.
+* Take a snapshot of your Elasticsearch and Kibana indices. For more information, see link:https://www.elastic.co/guide/en/elasticsearch/reference/7.6/snapshot-restore.html[Snapshot and restore] in the Elasticseach documentation. Snapshotting gives you a list of all your current indices as a way to ensure that all indices have been recreated after the upgrade.
```

Comment:
I'm not certain where this suggestion comes from but we should say "backup your data" not point them to snapshot api. My recollection is this requires a common storage destination for all the nodes and we have not tested it.  We are bound to be asked questions if we provide this statement.

@sichvoge We need to tread carefully with this

cc @lukas-vlcek

Reply:
That is correct, we do not point users to Snapshot API. Notice that your link points to version 7x which we are not at yet. We should be pointing to https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-snapshots.html or https://www.elastic.co/guide/en/elasticsearch/reference/5.6/modules-snapshots.html

However, it brings the question of what type of backup we recommend because some types  may not be really ideal. See the comment about copying Lucene data files.

> It is not possible to back up an Elasticsearch cluster simply by taking a copy of the data directories of all of its nodes. Elasticsearch may be making changes to the contents of its data directories while it is running, and this means that copying its data directories cannot be expected to capture a consistent picture of their contents. Attempts to restore a cluster from such a backup may fail, reporting corruption and/or missing files, or may appear to have succeeded having silently lost some of its data. The only reliable way to back up a cluster is by using the snapshot and restore functionality.

Reply:
So what's the alternative so that in case something did not work while upgrading, users can go back to a previous state?

Reply:
@sichvoge they have to go through a painful procedure of restoring their disks but if we have not tested snapshotting then we have no business advising its use.  We can not even declare if the cluster is properly set up to support snapshotting

Reply:
Currently at _Back up your Elasticsearch and Kibana data._